patroni/ctl.py

```diff
@@ -630,7 +719,23 @@ def members(config_file, cluster_names, fmt, watch, w, dcs):
         dcs = get_dcs(config, cluster_name)
 
         for _ in watching(w, watch):
-            output_members(dcs.get_cluster(), cluster_name, fmt)
+            extra = dict()
+            cluster = dcs.get_cluster()
+            if extended:
+                extra = {
+                    'Scheduled restart': dict()
+                }
+                for member in cluster.members:
+                    value = ''
+                    status = patroni_status(member, auth_header(config))
```

Comment:
Does it really make sense to get status of every node?
Information about scheduled restarts is available from DCS...
member.data.get('scheduled_restart')